db/db_test2.cc

```diff
@@ -1129,23 +1138,42 @@ TEST_F(DBTest2, DISABLED_PresetCompressionDict) {
       ASSERT_EQ(0, NumTableFilesAtLevel(0, 1));
       ASSERT_GT(NumTableFilesAtLevel(1, 1), 0);
 
-      size_t out_bytes = 0;
+      // Count the live sst files size
+      size_t all_sst_bytes = 0;
       std::vector<std::string> files;
-      GetSstFiles(env_, dbname_, &files);
+      uint64_t manifest_file_size = 0;
+      ASSERT_OK(db_->GetLiveFiles(files, &manifest_file_size));
```

Comment:
It's better to use `DB::GetColumnFamilyMetaData()`, which returns file size too, so you don't have to do the string operations and Env operations.

Reply:
Thanks for the tip. Switched to using `DBTestBase::TotalSize` now, which is based on `DB::GetColumnFamilyMetaData()`.